src/proto/mgmt/pool.proto

```diff
@@ -60,7 +60,9 @@ message PoolEvictReq {
 	string sys = 1; // DAOS system identifier
 	string id = 2; // uuid or label of pool to evict
 	repeated uint32 svc_ranks = 3; // List of pool service ranks
-	repeated string handles = 4; // Optional list of handles to evict
+	repeated string handles = 4; // Optional list of handles to evict (Mutually exclusive with destroy/force)
+	bool destroy = 5; // If true, evict is first step of a pool destroy operation
+	bool force = 6; // If true and destroy=true, forcibly closes open pool handles
```

Comment:
This seems a little confusing to me... It only really means force if it's also part of a destroy operation?

Reply:
Renamed to force_destroy